from error import *

def main():
    print("hi")
    raise SaveError(dump="a")